packs/kingmaker-bestiary/skeletal-tiger-lord.json

```diff
@@ -501,7 +501,7 @@
                 "category": "defensive",
                 "deathNote": true,
                 "description": {
-                    "value": "<p>When a skeletal Tiger Lord is destroyed, its bones clatter to the ground in a tangle and its armor and weapons crumble to dust. It automatically reforms with its armor and weapons intact (regaining a full quiver of 20 arrows) 1 hour later at 1 Hit Point, allowing its fast healing to resume healing it thereafter. If the trap in area D5 is disabled, the skeleton loses this ability; otherwise, the only way to permanently destroy a skeletal Tiger Lord is to reduce it to 0 Hit Points with vitality damage or evil damage.</p>"
+                    "value": "<p>When a skeletal Tiger Lord is destroyed, its bones clatter to the ground in a tangle and its armor and weapons crumble to dust. It automatically reforms with its armor and weapons intact (regaining a full quiver of 20 arrows) 1 hour later at 1 Hit Point, allowing its fast healing to resume healing it thereafter. If the trap in area D5 is disabled, the skeleton loses this ability; otherwise, the only way to permanently destroy a skeletal Tiger Lord is to reduce it to 0 Hit Points with vitality damage or holy damage.</p>"
```

Comment:
```suggestion
                    "value": "<p>When a skeletal Tiger Lord is destroyed, its bones clatter to the ground in a tangle and its armor and weapons crumble to dust. It automatically reforms with its armor and weapons intact (regaining a full quiver of 20 arrows) 1 hour later at 1 Hit Point, allowing its fast healing to resume healing it thereafter. If the trap in area <strong>D5</strong> is disabled, the skeleton loses this ability; otherwise, the only way to permanently destroy a skeletal Tiger Lord is to reduce it to 0 Hit Points with vitality damage or a holy effect.</p>"
```
Maybe that works?